core/modules/widgets/action-listops.js

```diff
@@ -3,87 +3,89 @@ title: $:/core/modules/widgets/action-listops.js
 type: application/javascript
 module-type: widget
 
-Action widget to apply list operations to any tiddler field (defaults to the 'tags' field of the current tiddler)
+Action widget to apply list operations to any tiddler field (defaults to the 'list' field of the current tiddler)
 
 \*/
 (function () {
 
-  /*jslint node: true, browser: true */
-  /*global $tw: false */
-  "use strict";
+    /*jslint node: true, browser: true */
+    /*global $tw: false */
+    "use strict";
 
-  var Widget = require("$:/core/modules/widgets/widget.js").widget;
+    var Widget = require("$:/core/modules/widgets/widget.js")
+        .widget;
 
-  var ActionListopsWidget = function (parseTreeNode, options) {
-    this.initialise(parseTreeNode, options);
-  };
+    var ActionListopsWidget = function (parseTreeNode, options) {
+        this.initialise(parseTreeNode, options);
```

Comment:
Confused? I've withdrawn a PR by deleting the branch -- however, when I make a new PR with tabs instead of spaces, I still get diffs for most of the lines of code. It appears that the original file neither used tabs nor 4 spaces.

Which code formatter is recommended for submitting to TW5?